src/actions/__tests__/bitcoinActions.test.js

```diff
@@ -51,21 +51,23 @@ describe('Bitcoin actions', () => {
     store = mockStore(initialState);
   });
 
-  describe('refreshAddressBalanceAction', () => {
+  describe('refreshBitcoinBalanceAction', () => {
     describe('for existing address', () => {
       it('updates the balance', async () => {
-        await store.dispatch(refreshAddressBalanceAction(address, false));
+        return store.dispatch(refreshBitcoinBalanceAction(false))
```

Comment:
Why you didn't like the linear approach with `await` as it was before?
It looks weird that you're using async/await inside `.then()`

Reply:
I was having some issues but it turns out they weren't caused by this. Reverted.